### DISCUSSION TITLE
Using vite-plugin-icons-spritesheet in a monorepo?

### DISCUSSION THREAD

**Discussion by jam-fran** [forge42dev/vite-plugin-icons-spritesheet#14] [Q&A]
Before I start shoutout to @AlemTuzlak for this very handy package!

This might be an obvious question, but I'm having trouble wrapping my mind around the best way to use this plugin in a monorepo.

 For example, I'd like to have my icons in my internal "ui" package. But my ui package isn't compiled, that's handled by the apps within my monorepo that consume it (e.g. a remix app). So my ui package doesn't use vite, of course.
 
That may mean that this isn't the right solutoin for that use case, but I'm wondering if maybe the _parent_ app can utilize this package, or if there's some other workaround given those constraints. Any ideas would be very much appreciated!

**Comment by AlemTuzlak** [comment]
@jam-fran What we've done with our monorepo setup is we have a Storyblok instance that we run in the UI (which uses Vite) and that uses the plugin to auto-update the spritesheet which is registered in the package.json exports field and then consumed by other apps, I think it would maybe even be possible to register the path as "../../path" and have it listen to the ui but I haven't tried that to be honest

**Reply by PontusHorn** [reply]
Hey @AlemTuzlak, sorry to revive an old comment - this is what I'm trying to do as well, but running into issues with hot reload not triggering when adding/removing icons. I'm guessing because the icons are not within the app's root folder and not part of the module graph. Have you managed to solve this?
